### PR TITLE
Add all fields to directory serializer for admin, and added random user endpoint for fun

### PR DESCRIPTION
**Pull request by elimelt**

### CODE REVIEW

File: server/directory/serializers.py [hunk 1]
```diff
@@ -1,40 +1,61 @@
...
+            "linkedin",
+            "github",
+            "leetcode",
+        ]
+
+
+class AdminDirectoryMemberSerializer(BaseDirectoryMemberSerializer):
+    class Meta:
+        model = User
+        fields = "__all__"
```
Comment (ShawnCollinge) [inline server/directory/serializers.py:61]: Will this return the persons password?

File: server/directory/serializers.py [hunk 2]
```diff
@@ -29,12 +25,22 @@ def to_representation(self, instance):
...
+
+class RegularDirectoryMemberSerializer(BaseDirectoryMemberSerializer):
+    class Meta:
+        model = User
+        fields = ['id', 'username', 'first_name', 'last_name', 'major', 'grad_date', 'linkedin', 'github', 'leetcode']
+
+class AdminDirectoryMemberSerializer(BaseDirectoryMemberSerializer):
+    class Meta:
+        model = User
+        fields = '__all__'
```
[the remark this answers is not in the document]
Comment (elimelt) [reply server/directory/serializers.py]: salted and hashed yes

[the remark this answers is not in the document]
Comment (ShawnCollinge) [reply server/directory/serializers.py]: I don't think we should return users passwords at all.

[the remark this answers is not in the document]
Comment (elimelt) [reply server/directory/serializers.py]: updated

**Review by ShawnCollinge — APPROVED** [review]
LGTM

**Pull request merged**